src/App.tsx

```diff
@@ -1,14 +1,50 @@
 import "@buildo/bento-design-system/index.css";
 import "@buildo/bento-design-system/defaultTheme.css";
 import { defaultMessages } from "@buildo/bento-design-system/defaultMessages/en";
-import { BentoProvider, Title } from "@buildo/bento-design-system";
+import {
+  BentoProvider,
+  Headline,
+  Box,
+  ContentWithSidebar,
+  Body,
+} from "@buildo/bento-design-system";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import './i18n';
+import { useTranslation } from "react-i18next";
+import Home from "./pages/Home";
+
+
 
 function App() {
+  const { t } = useTranslation();
   return (
     <BentoProvider defaultMessages={defaultMessages}>
-      <Title size="large">Hello, World!</Title>
+      <BrowserRouter>
+        <Headline size="large" align="center">
+          {t("title")}
+        </Headline>
+
+        <ContentWithSidebar
+          sidebarPosition="left"
+          sidebarWidth="1/5"
+          sidebarBackground="backgroundOverlay"
+        >
+          <Box
+            height="full"
+            display="flex"
+            justifyContent="center"
+            alignItems="center"
+          >
+            <Body size="large">filters</Body>
+          </Box>
+          <Routes>
+            <Route path="/" element={<Home />} />
+            {/* <Route path="/" element={<Restaurant />} /> */}
+          </Routes>
+        </ContentWithSidebar>
```

Comment:
An observation of the routing structure: right now, if you were to navigate to a different route, you would still see the sidebar with the filters, but it wouldn't make much sense to have that in a detail view. I would move the sidebar structure inside the `Home` component.